### ISSUE TITLE
About process_data.py

### ISSUE THREAD

**Issue by mqwfrog** [paularindam/adaptive_weight_timeseries#1]
I noticed that datasets provided by the UCR archive only include ".txt", ".ts" and ".arff" files, while do not have ".tsv" files as used in your process_data.py. Maybe you should change ".tsv" to ".txt"?

**Comment by mqwfrog** [comment]
> I noticed that datasets provided by the UCR archive only include "txt", ".ts" and ".arff" files, while do not have ".tsv" files as used in your process_data.py. Maybe you should change ".tsv" to ".txt"?

**Comment by paularindam** [comment]
HiThis is not my repository. I cloned someone’s repository from a source code outside github hence it doesn’t show the clone information. I didn’t end up using this code. Also I thought this was a private repository. Maybe I mistakenly have it public. Sent from my iPhoneOn Dec 19, 2022, at 7:28 AM, mqwfrog ***@***.***> wrote:﻿
I noticed that datasets provided by the UCR archive only include ".ts" and ".arff" files, while do not have ".tsv" files as used in your process_data.py.
If I change ".ts" to ".tsv", there is the error: pandas.errors.ParserError: Error tokenizing data. C error: Expected 5 fields in line 8, saw 7.
So where can I get ".tsv" files?
